frontend/src/components/Modal.js

```diff
@@ -13,7 +13,9 @@ function Modal({ status, id }) {
           <div className="modal-body">{status}</div>
 
           <div className="modal-footer">
-            {id !== 'categoryAddModal' && status === 'Sikeres hozzáadás!' ? (
+            {id !== 'categoryAddModal' &&
```

Comment:
Ez a sor nem kell. Ha az id = recipeSuccess, akkor biztos nem egyenlo a categoryAdd-al.

Reply:
Nekem úgy nem múködik, még pluszt is kellett rakni hozzá

            {id !== 'categoryAddModal' &&
            id === 'recipeSuccessStatusModal' &&
            status === 'Sikeres hozzáadás!' ? (